src/pages/cropper.astro

```diff
@@ -155,7 +155,7 @@ import Base from "../components/base.astro";
 	function showToolbar(buttonX: number, buttonY: number) {
 		invoke('show_toolbar', {
 			buttonCoords: [buttonX, buttonY],
-			area: [startX, startY, endX, endY]
+			area: [startX + 2, startY + 2 , endX - 7, endY - 7]
```

Comment:
Nit: Can you write a comment on why these magic numbers were chosen?

Reply:
Picked the numbers using some trial and errors to remove the border from preview and final gif